LibSWBF2/Types/List.cpp

```diff
@@ -172,6 +173,19 @@ namespace LibSWBF2::Types
 		return *this;
 	}
 
+//TEMPFIX: see header
+#ifdef __clang__
+	template<class T>
+	const T& List<T>::operator[](const unsigned long i) const
```

Comment:
Lol is it really that he demands just a single operator[] that takes a `const unsigned long` as index and nothing else?

Reply:
I can't remember at the moment, it was a rather barbaric fix I made to avoid making the pull request any larger.  I'll look into the exact error tomorrow

Reply:
It's not a babaric fix, looks more like a necessity you needed to make to satify clangs specific needs